cpp/include/raft/sparse/linalg/detail/symmetrize.cuh

```diff
@@ -104,7 +104,7 @@ RAFT_KERNEL coo_symmetrize_kernel(int* row_ind,
       // Note that if we did find a match, we don't need to
       // compute `res` on it here because it will be computed
       // in a different thread.
-      if (!found_match && vals[idx] != 0.0) {
+      if (!found_match && cur_val != 0.0) {
```

Comment:
I'm a little apprehensive about this. This is changing the actual value of this... are we sure this is correct?